- Use FreeBSD port version for RSTUDIO_VERSION
- Set FreeBSD-specific build ID

--- src/cpp/session/session-config.h.in.orig	2021-09-22 13:58:50 UTC
+++ src/cpp/session/session-config.h.in
@@ -13,7 +13,7 @@
  *
  */
 
-#define RSTUDIO_VERSION "${CPACK_PACKAGE_VERSION}"
+#define RSTUDIO_VERSION "${FREEBSD_RSTUDIO_VERSION}"
 #define RSTUDIO_VERSION_MAJOR "${CPACK_PACKAGE_VERSION_MAJOR}"
 #define RSTUDIO_VERSION_MINOR "${CPACK_PACKAGE_VERSION_MINOR}"
 #define RSTUDIO_VERSION_PATCH "${CPACK_PACKAGE_VERSION_PATCH}"
@@ -21,8 +21,8 @@
 #define RSTUDIO_RELEASE_NAME "${RSTUDIO_RELEASE_NAME}"
 #define RSTUDIO_BUILD_TYPE "${RSTUDIO_BUILD_TYPE}"
 #define RSTUDIO_GIT_REVISION_HASH "${RSTUDIO_GIT_REVISION_HASH}"
-#define RSTUDIO_GIT_COMMIT "${RSTUDIO_GIT_COMMIT}"
-#define RSTUDIO_BUILD_ID "${RSTUDIO_BUILD_ID}"
+#define RSTUDIO_GIT_COMMIT "n/a"
+#define RSTUDIO_BUILD_ID "FreeBSD-port-version-${FREEBSD_RSTUDIO_VERSION}"
 #define RSTUDIO_BUILD_DATE "${CPACK_BUILD_DATE}"
 #define RSTUDIO_COPYRIGHT_YEAR "${CPACK_COPYRIGHT_YEAR}"
 #define RSTUDIO_PACKAGE_OS "${RSTUDIO_PACKAGE_OS}"
